lib/mongoid/criteria.rb

```diff
@@ -216,10 +218,12 @@ def initialize(klass)
     #
     # @since 2.2.0
     def includes(*relations)
```

Comment:
`#includes` method needs doc updated for `relations` param to be Array<Symbol **or Hash**>